### PR TITLE
docs: adds context menu accessibility

### PR DESCRIPTION
**Pull request by lindseywild**
Closes https://github.com/github/accessibility/issues/1266.

Adds documentation about custom context menus and accessibility.

### CODE REVIEW

File: content/accessibility/context-menus.mdx [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Context menus
+---
+
+## Overview
```
Comment (lindseywild) [inline content/accessibility/context-menus.mdx:5]: Would it be helpful to add a screenshot of a context menu?

**Review by owenniblock — APPROVED** [review]
Looks good to me. Agree that a screenshot would be helpful.

I also wonder if some visual examples of the indication would help? (I was not sure exactly what you mean by "A visual chevron" for example.

These are minor though - I approve! ❤️

File: content/accessibility/context-menus.mdx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Some web apps have complex functionality and having a custom context menu would help surface common options for a particular element. An example would be a spreadsheet application. Each cell may have multiple actions that could be taken, such as: Cut, Copy, Paste, Insert row, Delete row, etc. These actions may also have keyboard shortcuts associated with them, so having them in a menu will help a user understand what they are.
+
+## Guidelines
+
+All [semantics and functionality for accessible menus](https://www.w3.org/WAI/ARIA/apg/patterns/menu/) (for designers and engineers) apply to custom context menus.
+
+### For designers
+
+1. Unless there is a good use case (such as the spreadsheet use case previously mentioned), users may not be expecting the default context menu to be overwritten. Proceed with caution.
```
Comment (yaili) [inline content/accessibility/context-menus.mdx:19]: overwritten or overridden? 🤔

File: content/accessibility/context-menus.mdx (hunk 1 above)
Comment (yaili) [reply content/accessibility/context-menus.mdx:5]: Please make sure the image follows the same style as other images: https://primer.style/design/accessibility/tooltip-alternatives

**Review by bolonio — APPROVED** [review]

**Review by inkblotty — APPROVED** [review]
🚀 Great write-up. Thank you

File: content/accessibility/context-menus.mdx (hunk 1 above)
Comment (ichelsea) [reply content/accessibility/context-menus.mdx:5]: @yaili it's okay, I've got this one ;)

**Pull request closed without merging**